ddl/ddl_api.go

```diff
@@ -3363,6 +3381,17 @@ func (d *ddl) AlterTable(ctx context.Context, sctx sessionctx.Context, stmt *ast
 						Name: model.NewCIStr(opt.StrValue),
 					}
 				case ast.TableOptionEngine:
+				case ast.TableOptionTTL:
+					var ttlInfo *model.TTLInfo
+					ttlInfo, err = buildTTLInfo(opt)
+					if err != nil {
+						return err
+					}
+					err = d.AlterTableTTLInfo(sctx, ident, ttlInfo)
+				case ast.TableOptionTTLEnable:
+					err = d.AlterTableTTLEnable(sctx, ident, opt.BoolValue)
+				case ast.TableOptionNoTTL:
+					err = d.AlterTableTTLInfo(sctx, ident, nil)
```

Comment:
We should check remove TTL should not be together with alter TTL.

Reply:
Done

Reply:
After modifying to a standalone `ALTER REMOVE TTL` spec, multi-schema change is not supported by default. I add a test case to ensure that.